source/operations/install-deploy-manage/expand-minio-tenant.rst

```diff
@@ -55,44 +61,83 @@ Persistent Volumes
+
+The MinIO Operator supports expanding a MinIO Tenant by adding additional pools.
+
+.. tab-set::
+
+   .. tab-item:: Kustomization
+
+      #. Review the Kustomization object which describes the Tenant object (``tenant.yaml``).
+
+         The ``spec.pools`` array describes the current pool topology.
+
+      #. Add a new document to the ``spec.pools`` array.
+
+         The new pool must reflect your intended combination of Worker nodes, volumes per server, storage class, and affinity/scheduler settings.
+         See :ref:`minio-operator-crd` for more complete documentation on Pool-related configuration settings.
+
+      #. Apply the updated Tenant configuration
+
+         Use the ``kubectl apply`` command to update the Tenant:
+
+         .. code-block:: shell
+
+            kubectl apply -k ~/kustomization/TENANT-NAME
+
+         Modify the path to the Kustomization directory to match your local configuration.
+
+   .. tab-item:: Helm
+
+      #. Review the Helm ``values.yaml`` object.
+
+         The ``tenant.pools`` array describes the current pool topology.
+
+      #. Add a new document to the ``tenant.pools`` array.
+
+         The new pool must reflect your intended combination of Worker nodes, volumes per server, storage class, and affinity/scheduler settings.
+         See :ref:`minio-tenant-chart-values` for more complete documentation on Pool-related configuration settings.
+
+      #. Apply the updated Tenant configuration
+
+         Use the ``helm upgrade`` command to update the Tenant:
+
+         .. code-block:: shell
+
+            helm upgrade TENANT-NAME minio-operator/tenant -f values.yaml -n TENANT-NAMESPACE
+
+         The command above assumes use of the MinIO Operator Chart repository.
+         If you installed the Chart manually or by using a different repository name, specify that chart or name in the command.
 
-The MinIO Operator Console supports expanding a MinIO Tenant by adding additional pools.
+         Replace ``TENANT-NAME`` and ``TENANT-NAMESPACE`` with the name and namespace of the Tenant respectively.
+         You can use ``helm list -n TENANT-NAMESPACE`` to validate the Tenant name.
 
+You can use the ``watch kubectl get all -n TENANT-NAMESPACE`` to monitor the progress of expansion.
+The MinIO Operator updates services to route connections appropriately across the new nodes.
+If you use customized services, routes, ingress, or similar Kubernetes network components, you may need to update those components for the new pod hostname ranges.
 
-1) Expand the MinIO Tenant
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. Following link is intended for K8s only
+.. _minio-decommissioning:
 
-#. From the Operator Console view, select the Tenant to open the summary view, then select :guilabel:`Pools`.
-   Select :guilabel:`Expand Tenant`.
+Decommission a Tenant Server Pool
+----------------------------------
 
-#. Specify the following information for the new pool:
+Decommissioning a server pool involves three steps:
```

Comment:
Four.
They'll also need to manually delete the decommissioned storage. See [operator/PR#2203](https://github.com/minio/operator/pull/2203).